### DISCUSSION TITLE
Welcome to Interclip mobile Discussions!

### DISCUSSION THREAD

**Discussion by filiptronicek** [interclip/mobile#219] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by bigint** [comment]
Yayy 🎉
